engine/src/main/java/org/terasology/logic/behavior/BehaviorSystem.java

```diff
@@ -102,6 +102,12 @@ public void update(float delta) {
         Iterable<EntityRef> entities = entityManager.getEntitiesWith(BehaviorComponent.class);
         for (EntityRef entity : entities) {
             BehaviorComponent behaviorComponent = entity.getComponent(BehaviorComponent.class);
+            // NPE observed in the past, suspected to be about loss of behavior state. Hopefully one skip is OK then resume next tick?
+            // TODO: Highlight this log entry to the telemetry system to gather better data over time
+            if (behaviorComponent.interpreter == null) {
```

Comment:
Oh, the behavior stuff is still part of the engine o.O

Reply:
I don't really remember why that particular piece is still in there 🤔